### PR TITLE
belindas-closet-nextjs_5_294_Dashboard Page

### PR DESCRIPTION
**Pull request by BradleyCharles**
Resolves issue #294 

Added a landing dashboard page with a link in the navbar.

dashboard1
dashboard2

### CODE REVIEW

**Review by alasali1 — COMMENTED** [review]
It looks good to me. I was able to see the dashboard button, and navigate to the dashboard page successfully.

image

image

**Review by alasali1 — APPROVED** [review]

**Comment by nwm516** [comment]
It worked for me for the most part when replicating it in my local environment. However, I did notice some routing issues when navigating to the Dashboard from certain pages.


https://github.com/SeattleColleges/belindas-closet-nextjs/assets/36825393/b5092cdb-fe38-4b83-bbf3-98b7983a0cba

Certain pages seemed to get there just fine, whereas others had some unexpected output, as you can see. Probably comes down to routing issues that may extend beyond the scope of your user story but felt it was worth mentioning. I imagine that since the Dashboard will be an exclusive feature to Admins that these issues may be moot, in the long run.

**Comment by cshimm** [comment]
It looks like the 404 is happening because of the missing '/' in front of 'dashboard' in the links array. adding that fixed that 404 problem 
image

**Review by taylorpapke — APPROVED** [review]
Looks great!
Screenshot 2024-04-22 at 2 40 21 PM

File: app/dashboard/page.module.css
Comment (tinpham5614) [inline app/dashboard/page.module.css:1]: Since we are using MUI for styling, we may not require a CSS file.

File: components/Navbar.tsx [hunk 1]
```diff
@@ -21,8 +21,14 @@ import MenuIcon from "@mui/icons-material/Menu";
...
 const drawerWidth = 240;
-const navItems = ["Home", "Sign In", "Donation", "Mission"];
-const links = ["/", "/auth/sign-in", "/donation-info", "/mission-page"];
+const navItems = ["Home", "Sign In", "Donation", "Mission", "Dashboard"];
+const links = [
+  "/",
+  "/auth/sign-in",
+  "/donation-info",
+  "/mission-page",
+  "dashboard",
```
Comment (tinpham5614) [inline components/Navbar.tsx:30]: There seems to be a routing error due to a missing forward slash "/". Adding the forward slash should resolve the issue.

File: app/dashboard/page.module.css
Comment (tinpham5614) [inline app/dashboard/page.module.css:1]: Since we are using MUI for styling, we may not require a CSS file.

**Review by tinpham5614 — CHANGES_REQUESTED** [review]

File: app/dashboard/page.module.css
Comment (BradleyCharles) [reply app/dashboard/page.module.css:1]: I removed the CSS file as it was not needed

**Comment by tinpham5614** [comment]
There are conflicts due to codes are not up-to-date.

**Comment by brinkbrink** [comment]
Looks like there are issues with signing commits here. Pro-tip: You also have to remember to sign the commits after you've created your key. It may be helpful to look into auto-signing your commits (which is what I do).

Here is a good bit of advice from [GitHub's docs](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits):

To configure your Git client to sign commits by default for a local repository, in Git versions 2.0.0 and above, run `git config commit.gpgsign true`. To sign all commits by default in any local repository on your computer, run `git config --global commit.gpgsign true`.

To store your GPG key passphrase so you don't have to enter it every time you sign a commit, we recommend using the following tools:

For Mac users, the [GPG Suite](https://gpgtools.org/) allows you to store your GPG key passphrase in the macOS Keychain.
For Windows users, the [Gpg4win](https://www.gpg4win.org/) integrates with other Windows tools.
You can also manually configure [gpg-agent](http://linux.die.net/man/1/gpg-agent) to save your GPG key passphrase, but this doesn't integrate with macOS Keychain like ssh-agent and requires more setup.

**Pull request merged**